### ISSUE TITLE
Cyrillics small caps

### ISSUE THREAD

**Issue by landswellsong** [libertinus-fonts/libertinus#44]
Seems to be missing. Is there any way to make them in bulk? (0 experience editing fonts here)

**Comment by khaledhosny** [comment]
No shortcuts, unfortunately. They will have to be drawn and by someone who knows Cyrillic (i.e. not me).

**Comment by ArTourter** [comment]
For the Russian part of cyrillic, it shouldn't involve too many new glyphs as most of them will be identical to their lower case version. The ones that need to be created should be:
а б е р с у ф (and their accentuated companions such as ё), and even then, most of then will be identical to the latin glyphs (except б y and ф)

Hope this helps

**Comment by KrasnayaPloshchad** [comment]
I also think ђ, ј, ћ, ў ahould also add their own small caps.

**Comment by KrasnayaPloshchad** [comment]
These letters also need this variant: ѐ, ё, і, ї, ј, ӕ, ӯ, ӱ, ӳ

**Comment by moyogo** [comment]
Is there any reason all cyrillic letters shouldn’t have small caps?

**Comment by moyogo** [comment]
Let me rephrase that: Is there any reason only some cyrillic should have small caps and not all?

**Comment by KrasnayaPloshchad** [comment]
The implementation of Cyrillic small caps should exclude several Church Slavonic letters (ѧ, ѫ, ѯ, ѱ, ...), because Church Slavonic texts does not use western styled typeface in general, many other archaic Slavonic texts also use the same face, they has their own traditions for typography differ from the Western (see [UTN No.41](http://www.unicode.org/notes/tn41/) for details), until Peter the Great’s [reform](https://en.wikipedia.org/wiki/Reforms_of_Russian_orthography#18th-century_changes) in Russia, in which he introduced westernized letter forms and abolished some letters as above, then other languages that use the script also follow this action, since then, only Church Slavonic used old typeface and orthography.
![An example of Peter the Great’s reform](https://upload.wikimedia.org/wikipedia/commons/7/7a/Russian_alphabet_%28marks_by_Peter_I%29%2C_page_5.gif)
If you want to add Church Slavic texts and other archaic Slavic texts in your document, it’s better using specific font for text rendering, for example, [Ponomar Unicode](http://sci.ponomar.net/fonts.html).

**Comment by skef** [comment]
I've been looking at Small caps-related issues. 

As with #154 , it appears that the cryillic smallcap glyphs are in LinLibertine_C-4.0.4.sfd. So (setting more recent changes aside) this should be mostly a matter of pulling and merging the glyphs rather than designing new ones. 

However, there are many accented variants, posing the same problem as #78 . This raises the question of what to do about merging all of these accented characters in relation to the "redo the mark system from scratch" option in #67 . One option would be to research and implement the new mark system with smallcaps first, importing the extra glyphs as part of that project, and then go back and implement that system with the other glyphs. That way there is a "test bed" with a font variant that is effectively broken at present anyway.

**Comment by khaledhosny** [comment]
Not doing anything here. People who needs this or know what exactly is needed here are free to send pull requests.

**Comment by den32ds** [comment]
@khaledhosny yeah, Cyrillic and Greek small caps present in older versions Linux Libertine, but I do not have enough experience to prescribe them OpenType functions. (P. S. Biolinum have small caps?)

**Comment by den32ds** [comment]
yeah, have
